### DISCUSSION TITLE
Add text to horizontal separator

### DISCUSSION THREAD

**Discussion by esurijon** [skrysmanski/chrome-separators#13] [Ideas]
Include a text box to add a custom text to be included in the link label, so the result goes as follows:

───── My Text ─────

**Comment by JustAFridg** [comment]
Hey @esurijon your idea is okay but you can do it your self aswell, its not that hard. Just right-click and then edit, then go in the middle of the ------ and then type what you want. 
![image](https://github.com/skrysmanski/chrome-separators/assets/123732650/02bd8a74-c9a8-4582-96e6-c4285cb8d939)
Just like that
*edit* 
But still like your idea, upvoted

**Comment by skrysm** [comment]
Thanks for the suggestion. I've converted this discussion into an issue (#19).
